documentation/related_projects/index.markdown

```diff
@@ -83,3 +83,16 @@ MoveGroupCapability that does simple extrapolation to check for possible collisi
 The [Nishida Takeshi Lab](http://lab.cntl.kyutech.ac.jp/) in Fukuoka, Japan supports a suite of ROS packages for working with the Motoman line of Robots from [Yaskawa](https://www.motoman.com/) through a number of interfaces including Matlab. You can explore some of these interfaces in the [motoman_apps](https://github.com/ntl-ros-pkg/motoman_apps) repository.
 
 <img src="/assets/images/related_projects/motoman_apps.png" height="500"/>
+
+## Cartesian Path Planner Plug-In for MoveIt!
+
+[ROS Industrial](https://rosindustrial.org/) create a user friendly [Cartesian Path Planner Plug-In for MoveIt!](https://rosindustrial.org/news/2014/9/5/cartesian-path-planner-plug-in-for-moveit). In the current version of the project, the user can simultaneously interact with a Qt Widget and the RViz environment to define and set Cartesian Way-Points, which can then be passed to the Cartesian Planner of the MoveIt package and executed both on a simulated and real robot.
```

Comment:
also, we have "MoveIt!." and I don't like the double punctuation, but I am not sure of the best fix for that